lambdas/answer/src/main/java/uk/gov/di/ipv/cri/kbv/api/handler/QuestionAnswerHandler.java

```diff
@@ -62,10 +63,13 @@ public class QuestionAnswerHandler
     @ExcludeFromGeneratedCoverageReport
     public QuestionAnswerHandler() {
         ServiceFactory serviceFactory = new ServiceFactory();
+        DynamoDbEnhancedClient dynamoDbEnhancedClient = serviceFactory.getDynamoDbEnhancedClient();
```

Comment:
No need to hold the reference, we can pass `serviceFactory.getDynamoDbEnhancedClient()` directly into the creation of `KBVStorageService`. Same for the other bits, like how you've done it for `AbandonKbvHandler`